NorthstarDLL/squirrel.cpp

```diff
@@ -379,4 +379,28 @@ ON_DLL_LOAD_RELIESON("server.dll", ServerSquirrel, ConCommand, [](HMODULE baseAd
 		ConCommand_script<ScriptContext::SERVER>,
 		"Executes script code on the server vm",
 		FCVAR_GAMEDLL | FCVAR_GAMEDLL_FOR_REMOTE_CLIENTS | FCVAR_CHEAT);
-})
\ No newline at end of file
+})
+
+int GetReturnTypeEnumFromString(const char* name)
+{
+
+	static std::map<std::string, SQReturnTypeEnum> sqEnumStrMap = {
+		{"bool", SqReturnBoolean},
+		{"float", SqReturnFloat},
+		{"vector", SqReturnVector},
+		{"int", SqReturnInteger},
+		{"entity", SqReturnEntity},
+		{"string", SqReturnString},
+		{"array", SqReturnArrays},
+		{"asset", SqReturnAsset},
+		{"table", SqReturnTable}};
+
+	if (sqEnumStrMap.count(name))
+	{
+		return sqEnumStrMap[name];
+	}
+	else
+	{
+		return SqReturnDefault; // previous default value
```

Comment:
call me stupid but where is this actually defined?

Reply:
i mean, the build succeeded, so... search for it?

Reply:
It is part of the enum I defined in the header. Previously it was just set in the constructor of the SQFuncRegistration struct without a named value